annif/annif.py

```diff
@@ -62,9 +76,19 @@ def listprojects():
     doc = {'size': 1000, 'query': {'match_all': {}}}
     results = es.search(index=annif.config['INDEX_NAME'], doc_type='project',
                         body=doc)
+
     projects = [x['_source'] for x in results['hits']['hits']]
+
+    def parseRow(col1, col2, col3):
```

Comment:
Nitpick on naming: I would call this function `formatRow`, not `parseRow`. What it does is the opposite of parsing...